### DISCUSSION TITLE
S2 是否可以实现留存图

### DISCUSSION THREAD

**Discussion by yanyabing1** [antvis/S2#2433] [Q&A]
![image](https://github.com/antvis/S2/assets/22764669/49bec6fb-23ba-4725-8422-66c4206cb5a7)

**Comment by lijinke666** [comment]
这种自己用 dom 实现不是更方便吗? S2 理论上是可以的, 提供一种思路

- 通过条件格式自定义单元格背景 https://s2.antv.antgroup.com/examples/analysis/conditions/#multi-background
- 没有数据则让单元格背景和文本都变透明
- 关闭 hover/click 等内置交互
- 自定义单元格或配置主题, [重写边框](https://s2.antv.antgroup.com/zh/examples/case/art/#mosaic) (不绘制或者让边框变透明)
